docs/concepts/workloads/pods/podpreset.md

```diff
@@ -0,0 +1,79 @@
+---
+approvers:
+- jessfraz
+title: Pod Preset
+---
+
+{% capture overview %}
+This page provides an overview of PodPresets, which are objects for injecting
+certain information into pods at creation time. The information can include
+secrets, volumes, volume mounts, and environment variables.
+{% endcapture %}
+
+{:toc}
+
+{% capture body %}
+## Understanding Pod Presets
+
+A "Pod Preset" is an API resource for injecting additional runtime requirements
+into a Pod at creation time.
+You use [label selectors] (/docs/concepts/overview/working-with-objects/labels/#label-selectors)
+to specify the Pods to which a given Pod Preset applies.
+
+Using a Pod Preset allows pod template authors to not have to explicitly provide 
+all information for every pod. This way, authors of pod templates consuming a
+specific service do not need to know all the details about that service.
+
+See [PodPreset proposal](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md) for more information.
```

Comment:
This might be better referenced as "See the [design proposal for PodPreset]" for the link to the hypertext to make it clear it's the original design thinking. Not critical though